README.md

```diff
@@ -1,6 +1,14 @@
 # EveryPolitician Pull Request summary
 
-Produces a summary of an EveryPolitician data pull request like [this one](https://github.com/everypolitician/everypolitician-data/pull/16984#issuecomment-246282991).
+This is the code the EverPolitician bot uses to inspect data pull requests (specifically those are on the
```

Comment:
"those are on" seems wrong here. Probably should either be "those that are on" or just "those on"